packages/dso-toolkit/components/Componenten/rich-content/rich-content.njk

```diff
@@ -43,7 +43,7 @@
     </div>
   </dso-highlight-box>
   <p>Lorem Ipsum is slechts een proeftekst uit het drukkerij- en zetterijwezen. Lorem Ipsum is de standaard proeftekst in deze bedrijfstak sinds de 16e eeuw, toen een onbekende drukker een zethaak met letters nam en ze door elkaar husselde om een font-catalogus te maken.</p>
-  <p><img src="{{ '/dummy/images/ocean-480.jpg' | path }}" /></p>
+  <p><img src="{{ '/dummy/images/ocean-480.jpg' | path }}" alt="Afbeelding van een oceaangolf" /></p>
```

Comment:
Ik denk dat we hier best wel gebruik mogen/kunnen maken van ons bestaande `image` component (zoals ook op regel 49 gedaan is)